src/js/components/List.js

```diff
@@ -98,38 +99,51 @@ export default class List extends Component {
   }
 
   render () {
-    var classes = [CLASS_ROOT];
-    if (this.props.selectable) {
-      classes.push(CLASS_ROOT + "--selectable");
-    }
-    if (this.props.className) {
-      classes.push(this.props.className);
-    }
+    const {
+      children,
+      className,
+      emptyIndicator,
+      onMore,
+      role,
+      selectable
+    } = this.props;
+
+    const classes = classnames(
+      CLASS_ROOT,
+      className,
+      {
+        [`${CLASS_ROOT}--selectable`]: selectable,
+        [`${CLASS_ROOT}--moreable`]: onMore
+      }
+    );
 
     let empty;
-    if (this.props.emptyIndicator) {
+    if (emptyIndicator) {
       empty = (
-        <li className={CLASS_ROOT + "__empty"}>
-          {this.props.emptyIndicator}
+        <li className={`${CLASS_ROOT}__empty`}>
+          {emptyIndicator}
         </li>
       );
     }
 
-    var more;
-    if (this.props.onMore) {
-      classes.push(CLASS_ROOT + "--moreable");
+    let more;
+    if (onMore) {
       more = (
-        <li ref={(ref) => this.moreRef = ref} className={CLASS_ROOT + "__more"}>
+        <li ref={(ref) => this.moreRef = ref} className={`${CLASS_ROOT}__more`}>
           <SpinningIcon />
         </li>
       );
     }
 
     return (
-      <ul ref={(ref) => this.listRef = ref}
-        className={classes.join(' ')} onClick={this._onClick}>
+      <ul
+        ref={(ref) => this.listRef = ref}
+        className={classes}
+        onClick={this._onClick}
+        role={role}
```

Comment:
I don't think we need to explicitly declare "role" here, but transfer the props down to the ul component.

**Box implementation for transferring props**: https://github.com/grommet/grommet/blob/master/src/js/components/Box.js#L72

**Original issue tracking the problem**:

https://github.com/grommet/grommet/issues/85